vllm/config.py

```diff
@@ -415,6 +423,7 @@ def _get_and_verify_dtype(
     # NOTE: getattr(config, "torch_dtype", torch.float32) is not correct
     # because config.torch_dtype can be None.
     config_dtype = getattr(config, "torch_dtype", None)
+    print(f"dtype given from config is {config_dtype}")
```

Comment:
Debug print statement?

Reply:
Yeah I'm not at the cleaning/formatting stage just yet